docs/docs/sample-scripts/teams/export-teams-conversations.md

```diff
@@ -0,0 +1,95 @@
+# Export conversations from Microsoft Teams Channels
+
+Author: [Joseph Velliah](https://sprider.blog/export-conversations-from-microsoft-teams)
+
+## Problem Statement
+
+We utilize Teams during incidents and create channels for each. We would like to be able to export conversation history.
+
+- Right now the only option we have is to go through Security & Compliance.
+- Teams usage is growing in every organization and it would soon become unreasonably to only have Administrators be the ones doing exports of channels for all the Teams.
```

Comment:
This statement gives the impression that you can use this script to export any conversation even if you're not an admin. That's not the case, isn't it? We should adjust it to make it clear that you do need the necessary permissions to export conversations.

Reply:
As the team member or owner can not export the conversation via frontend(browser/client), I have mentioned this as a problem statement here. I have updated the solution section based on your feedback. Hope this will be appropriate. Please let me know if you need any changes.